packit_service/worker/handlers/abstract.py

```diff
@@ -463,3 +465,11 @@ def __init__(
 
     def run(self) -> TaskResults:
         raise NotImplementedError("This should have been implemented.")
+
+
+class FedoraCIJobHandler(JobHandler):
+    check_name: str = ""
+
+    @classmethod
+    def get_check_names(cls, service_config: ServiceConfig, metadata: EventData) -> list[str]:
```

Comment:
this is returning a list because of the future use case of running multiple testsuits?

Reply:
Yes, so far there is only one test/suite: https://github.com/packit/packit-service/blob/a8b2d13d55dc21591cc5e680b74d190090bfbccb/packit_service/worker/helpers/testing_farm.py#L1638